main.cc

```diff
@@ -1352,7 +1352,7 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 std::ref(feature_service), std::ref(mm), std::ref(token_metadata), std::ref(erm_factory),
                 std::ref(messaging), std::ref(repair),
                 std::ref(stream_manager), std::ref(lifecycle_notifier), std::ref(bm), std::ref(snitch),
-                std::ref(tablet_allocator), std::ref(cdc_generation_service)).get();
+                std::ref(tablet_allocator), std::ref(cdc_generation_service), std::ref(task_manager)).get();
```

Comment:
The storage_service-task_manager dependency deserves its own commit with description